src/lib/cimb_translator/Common.cpp

```diff
@@ -19,27 +20,53 @@ using std::vector;
 namespace {
 	RGB getColor4(unsigned index)
 	{
-		// opencv uses BGR, but we don't have to conform to this tyranny
+		// opencv uses BGR, but we don't have to conform to its tyranny
 		static constexpr array<RGB, 4> colors = {
-		    RGB(0, 0xFF, 0xFF),
-		    RGB(0xFF, 0xFF, 0),
-		    RGB(0xFF, 0, 0xFF),
-		    RGB(0, 0xFF, 0)
+			RGB(0, 0xFF, 0),
```

Comment:
Since we're changing the format anyway, I fixed the color ordering. In practice, because reedsolomon is fixing byte errors (not bit errors), this shouldn't change anything. But it does lead to less bit errors.

Reply:
As a higher level note, it might make sense to fold the dark/light distinction into these color sets. Targeting a white background will require a different color set anyway.